pkgs/development/python-modules/typing-extensions/default.nix

```diff
@@ -32,7 +32,8 @@ buildPythonPackage rec {
   ];
 
   meta = with lib; {
-    description = "Backported and Experimental Type Hints for Python 3.5+";
+    description = "Backported and Experimental Type Hints for Python";
+    changelog = "https://github.com/python/typing_extensions/blob/main/CHANGELOG.md";
```

Comment:
```suggestion
    changelog = "https://github.com/python/typing_extensions/blob/${src.rev}/CHANGELOG.md";
```

Reply:
Adopted in [force-push](https://github.com/NixOS/nixpkgs/compare/c34b82d178df53ac5beb6ae59a51644cef203971..538e8e2d1834ada53b51f7a31494d2b00fe8aa5a). Thx!